readthedocs/core/management/commands/contact_owners.py

```diff
@@ -74,6 +74,16 @@ def add_arguments(self, parser):
             '--notification',
             help='Path to a file with the notification content in markdown.',
         )
+        parser.add_argument(
+            '--sticky',
+            action='store_true',
+            dest='sticky',
+            default=False,
+            help=(
+                'If the notification should be sticky '
+                '(the notification disappears after the first view by default).'
```

Comment:
```suggestion
                'Make the notification sticky '
                '(the notification stays until the user closes it)'
```

I think it's better to describe how the argument works instead of describing how the default works when not passing `--sticky`.